main/src/io/ifile_writer.hpp

```diff
@@ -30,7 +30,7 @@ struct AsciiWriter : public IFileWriter<Dataset>
               std::string path) const override
     {
         const char separator = ' ';
-        path += std::to_string(d.iteration) + ".txt";
+        path += std::to_string(d.ttot) + ".txt";
```

Comment:
I think it's confusing to use floating point numbers in file names. An important use-case for these files
is to do quick checks and debugging on certain steps. If you insist on using plain text files for scientific experiments with time series, then you can find the iteration time in `constants.txt`

Reply:
I don't think it is confusing. It is in fact something that is commonly used mostly when you want outputs at specific times and you don't want to keep looking in the table which is which. I'd appreciate having both options if possible.